### ISSUE TITLE
about loss

### ISSUE THREAD

**Issue by LeopardYanZhang** [zylo117/Yet-Another-EfficientDet-Pytorch#677]
I use my 3080 GPU to train coco2017 from scratch for about 2 days
but the result is so strange

![image](https://user-images.githubusercontent.com/32701615/127765222-15fbc31e-8cb4-4298-9a1e-3df6abca5487.png)
![image](https://user-images.githubusercontent.com/32701615/127765230-619a0def-ac9f-43a0-9638-077ce3cde3ea.png)

I used the commond:
python train.py -c 0 --batch_size 8 --optim sgd --lr 5e-2

**Comment by zylo117** [comment]
5e-2 blows the roof off

**Comment by LeopardYanZhang** [comment]
> 5e-2 blows the roof off

thanks a lot.,After I changed my lr ,it does works.
WellI have another problem:how many days will be token if  I train the coco from scratch with a GPU 3080.
